src/lib/lumi.ts

```diff
@@ -2829,6 +2830,136 @@ export const fromZigbee = {
             }
         },
     } satisfies Fz.Converter,
+    lumi_curtain_speed: {
```

Comment:
Could you refactor these converters to modernExtend? This replaces the to/fromZigbee and exposes. Example: https://github.com/marthubner/zigbee-herdsman-converters/blob/2ca6323e34117f1686bdac9c98774397c25ece53/src/lib/lumi.ts#L1518

Reply:
I tried to do so